tests/darwin/client_test.py

```diff
@@ -0,0 +1,117 @@
+from pathlib import Path
+from typing import Any, Dict
+
+import pytest
+import responses
+from requests.exceptions import HTTPError
+
+from darwin.client import Client
+from darwin.config import Config
+from darwin.exceptions import InsufficientStorage, Unauthorized
```

Comment:
IMHO not so pythonic names, `InsufficientStorageError, UnauthorizedError` are better

Reply:
Yes, but I didn't create them, I am just going with the flow here :P